venues/cv-foundation.org/ECCV/2018/Conference/python/upload-areachairs.py

```diff
@@ -0,0 +1,38 @@
+import openreview
+import argparse
+import csv
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+
+with open(os.path.join(os.path.dirname(__file__),'../data/areachairs.csv')) as f:
+    reader = csv.reader(f)
+    reader.next()
+    members = []
+    for line in reader:
+        first = line[0].strip()
+        middle = line[1].strip()
+        last = line[2].strip()
+        email = line[3].lower().strip()
+        try:
+            profile = openreview.tools.create_profile(client, first = first, middle = middle, last = last, email = email)
```

Comment:
it shouldn't create any profile, because all the profiles should come from researcher. So, I think we should change the call from create_profile to get_profile by email